ansible/roles/satellite-configuration/tasks/main.yml

```diff
@@ -0,0 +1,9 @@
+---
+
+- include: 01_manifest.yml
+- include: 02_repository.yml
+- include: 03_content_view.yml
+- include: 04_lifecycle.yml
+- include: 05_activationkey.yml
```

Comment:
`include` is deprecated, please use `import_tasks`

Reply:
replaced include to import_tasks